Multiple packages

KnisterPeter opened the pull request:
Will fix #60

KnisterPeter committed:
refactor: prepares for multipackage repository

KnisterPeter committed:
coverage

KnisterPeter committed:
feat: extended tasks

* bootstrap (incl. dependency linking)
* reset
* run

ZauberNerd commented:
FYI, Greenkeeper doesn't support monorepos. Please put a couple of +1s into this thread: https://github.com/greenkeeperio/greenkeeper/issues/139 ;)

KnisterPeter commented:
Thanks @ZauberNerd. Done.

KnisterPeter committed:
fix: fixes core test

core test requires update for new directory layout. Its independent of that now

KnisterPeter committed:
feat: run packages commands in order of package dependencies

KnisterPeter committed:
Merge remote-tracking branch 'origin/master' into multiple-packages
